src/components/grid-aware/Modal/Modal.module.css

```diff
@@ -0,0 +1,45 @@
+.overlay {
+  background-color: rgba(0, 0, 0, 0.5);
+  bottom: 0;
+  left: 0;
+  overflow: auto;
+  position: fixed;
+  right: 0;
+  top: 0;
```

Comment:
I assume you tried other methods in completely covering the whole view of the browser with the darkened hue of the modal  but is there a shorter way to cover the whole screen than putting all bottom, top, left right positions to 0?

Reply:
I actually didn't try anything else because I had copied these styles over from the default ones that the library comes with. I'm not sure if there's a shorter way to cover it, since it's `position: fixed`. You still need the `top: 0` since the element itself is placed based on where its parent is, not based on the page. The `right` and the `bottom` could be replaced with `width: 100vw` and `height: 100vh`, which doesn't save us much because it's a one-for-one tradeoff.

You could possibly leave out `left:`, but it will only work if you don't have anything that pushes the modal's parent's position away from the left edge, which makes it a little bit riskier to do.

Overall, I think setting the position across all four dimensions is pretty much as compact and reliable as we're going to get.